### PR TITLE
ci: add L2 ovm tests

### PR DESCRIPTION
**Pull request by drptbl**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -128,6 +136,38 @@ jobs:
...
+            cd optimism-integration
+            docker-compose pull
+      - run:
+          name: Start chains
+          background: true
+          command: |
+            cd optimism-integration
+            ./up.sh
+      - cmd-wait-for-rpc
+      - cmd-wait-for-hardhat
```
Comment (justinjmoses) [inline .circleci/config.yml:162]: can we change the port to a parameter and pass it in and then we don't have two separate commands?

**Review by justinjmoses — CHANGES_REQUESTED** [review]
One change with a parameter for a ci command

File: .circleci/config.yml (hunk 1 above)
Comment (drptbl) [reply .circleci/config.yml]: @justinjmoses done!

**Review by justinjmoses — APPROVED** [review]
🙏🏼

**Pull request merged**